cmd/soci/commands/push.go

```diff
@@ -45,12 +46,17 @@ If multiple soci indices exist for the given image, the most recent one will be
 After pushing the soci artifacts, they should be available in the registry. Soci artifacts will be pushed only
 if they are available in the snapshotter's local content store.
 `,
-	Flags: append(append(append(commands.RegistryFlags, commands.LabelFlag), commands.SnapshotterFlags...),
+	Flags: append(append(append(append(
```

Comment:
Can you please add an integration test for `push`, involving multiple platforms?